library/src/main/java/com/chuckerteam/chucker/internal/support/DateJsonAdapter.kt

```diff
@@ -0,0 +1,22 @@
+package com.chuckerteam.chucker.internal.support
+
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import java.text.DateFormat
+import java.util.Date
+import java.util.Locale
+
+class DateJsonAdapter {
+
+    @ToJson
+    fun toJson(date: Date): String {
+        val dateFormat = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, Locale.US)
+        return dateFormat.format(date)
+    }
+
+    @FromJson
+    fun fromJson(date: String): Date {
+        val dateFormat = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, Locale.US)
+        return dateFormat.parse(date)!!
```

Comment:
I would prefer to declare that this function can throw an exception and try to avoid `!!`.

Reply:
Fixed, also added several test cases for it.